batch.R

```diff
@@ -170,8 +170,8 @@ for(i in 1:nrow(fileDF)) {
   #TODO: model metrics for non-rloadest models
   #combine into DF with row for each model
   #need to extract fitted model so rloadest functions can be used
-  metrics <- summarizeModel(getFittedModel(rloadest5param))
-  write.csv(x = metrics, file = file.path(outputFolder, constitName, "modelMetrics", paste(constitSite, "modelMetrics.csv", sep = "_")), row.names = FALSE)
+  metrics <- summarizeModel(getFittedModel(rloadest5param), constituent = constitName, site = constitSite)
```

Comment:
It's part of a larger, long-running strategy: all the `loadReg` functions are essentially just helpers to the `loadReg2` functions, which are the ones I want users to use. I extended that logic to this new function without thinking too hard about it myself, but neglected to tell you about it.